src/core.c/Exception.pm6

```diff
@@ -343,7 +343,7 @@ my class X::Pragma::NoArgs is Exception {
 }
 my class X::Pragma::CannotPrecomp is Exception {
     has $.what = 'This compilation unit';
-    method message { "$.what may not be pre-compiled" }
+    method message { "$.what cannot be pre-compiled and thus cannot be used in a Module" }
```

Comment:
I'll still +1 it regardless, but I'd suggest 'module' instead of 'Module'. I took a quick look in the docs, and while the keyword is 'Module', most uses of the word not at the beginning of a sentence are lowercase.

Reply:
Good point; fixed.